### ISSUE TITLE
data pre-processor

### ISSUE THREAD

**Issue by Majorsnower** [XueZ-phd/TFDet#8]
You masked the train and test images, then detect.  In your paper, you mainly told that your work reduced the FPs. Does that make sense?

**Comment by XueZ-phd** [comment]
It is a good question.
In the KAIST dataset, there are many incorrect labels of bounding boxes. Our data pre-processing approach is designed to address this issue. For example, regions with many persons or unclear edges of individuals are often incorrectly assigned the "People" class. While our detector is capable of identifying each person in crowded areas, the incorrect ground-truth labels cause correct predictions to be misclassified. This is problematic for evaluating the effectiveness of a detection algorithm. To mitigate this, we mask out all regions with incorrect labels.

We would like to emphasize that our detector, TFDet, is effective at reducing false positives in background areas, rather than overfitting to the KAIST dataset, which contains numerous incorrect labels.

We also recommend using our processed dataset to train a robust detection algorithm, as this will save valuable time and prevent overfitting to a flawed dataset.
